spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -4233,6 +4234,27 @@ def __in_block_comment(cursor):
         cursor3.endEditBlock()
         return True
 
+    def create_new_cell(self):
+        firstline = '#%% ' + self.get_line_separator()
+        endline = self.get_line_separator()
+        cursor = self.textCursor()
+        if self.has_selected_text():
+            self.extend_selection_to_complete_lines()
+            start_pos, end_pos = cursor.selectionStart(), cursor.selectionEnd()
+            endline = self.get_line_separator() + '#%%'
```

Comment:
if the suggestion above is correct then we should change this too:

```suggestion
            endline = self.get_line_separator() + '# %%'
```